verticles/graphql/src/main/java/com/gentics/mesh/graphql/filter/BinaryFilter.java

```diff
@@ -43,11 +49,11 @@ private BinaryFilter(String name, String description, GraphQLContext context) {
 	@Override
 	public List<FilterField<HibBinary, ?>> getFilters() {
 		List<FilterField<HibBinary, ?>> filters = super.getFilters();
-		filters.add(new MappedFilter<>(owner, "checksum", "Filters by SHA512 checksum", StringFilter.filter(),
+		filters.add(new MappedFilter<>(owner, "sha512sum", "Filters by SHA512 checksum", StringFilter.filter(),
 				content -> content == null ? null : content.getSHA512Sum()));
 //		filters.add(new MappedFilter<>(owner, "variants", "Filters by image variants", 
 //				ListFilter.imageVariantListFilter(context, "BINARY"),
-//				content -> content == null ? null : (Collection<HibImageVariant>) CommonTx.get().binaryDao().getVariants(content, context).list(), Pair.pair("variants", new JoinPart("IMAGEVARIANT", "value"))));
+//				content -> content == null ? null : (Collection<HibImageVariant>) CommonTx.get().imageVariantDao().getVariants(content, context).list(), Pair.pair("variants", new JoinPart("IMAGEVARIANT", "value"))));
```

Comment:
ImageVariant filtering requires extra work on a native level, which is out of scope here.